src/mpfao/3D/tdympfao3D_core.c

```diff
@@ -38,7 +38,88 @@ PetscErrorCode TDyComputeEntryOfGMatrix3D(PetscReal area, PetscReal n[3],
 }
 
 /* -------------------------------------------------------------------------- */
-PetscErrorCode TDyComputeGMatrixFor3DMesh(TDy tdy) {
+PetscErrorCode TDyComputeGMatrixDefaultFor3DMesh(TDy tdy) {
```

Comment:
Is it worth documenting how this function differs from `TDyComputeGMatrixFor3DMesh`?

Reply:
Or maybe there's a better name than "Default". Unless everyone already knows that that means. :-)